manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "search-resolver",
-  "version": "0.16.13",
+  "version": "0.17.0",
```

Comment:
Why is this changing the node builder?

Reply:
I also wondered this, but it appears that's the Node builder is already on `6.x`, if you check this file.

Reply:
This will override the current version (that's running in node `7.x` and it's not merged in master)